README.md

```diff
@@ -1,10 +1,112 @@
 # Akka Persistence Postgres
 
 [![License](https://img.shields.io/:license-Apache%202-red.svg)](https://www.apache.org/licenses/LICENSE-2.0.txt)
+[![Actions Status](https://github.com/SwissBorg/akka-persistence-postgres/workflows/Scala%20CI/badge.svg)](https://github.com/SwissBorg/akka-persistence-postgres/actions)
 
 This is a fork of the official Akka's JDBC plugin, aimed to support PostgreSQL 11 features such as partitions, arrays, BRIN indexes and others. 
+It is a fork of the official Akka's JDBC plugin, aimed to support PostgreSQL 11 features such as partitions, arrays, BRIN indexes and others.
+All of this to reduce RAM usage and index size by the database.  
 
 ## Documentation
+## How to enable Akka Persistence Postgres in your project
 
 * [Akka Persistence JDBC 3.5.x documentation](https://doc.akka.io/docs/akka-persistence-jdbc/3.5/)
 * [Snapshot documentation](https://doc.akka.io/docs/akka-persistence-jdbc/snapshot/)
+To use this plugin prior default one, add the following to application.conf:
+```hocon
+akka.persistence {
+  journal.plugin = "postgres-journal"
+  snapshot-store.plugin = "postgres-snapshot-store"
+}
+```
+and for persistence query
+```scala
+PersistenceQuery(system).readJournalFor[JdbcReadJournal](JdbcReadJournal.Identifier)
```

Comment:
I'll rename it later when I change the name for the *Journal classes. This is okay for now 👍